docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1725,7 +1725,18 @@ Note: If you would like to move
 
 - Rename the exported `Omit` type in `@material-ui/types`. The module is now called `DistributiveOmit`. The change removes the confusion with the built-in `Omit` helper introduced in TypeScript v3.5. The built-in `Omit`, while similar, is non-distributive. This leads to differences when applied to union types. [See this StackOverflow answer for further details](https://stackoverflow.com/a/57103940/1009797).
 
+  ```diff
+  -import { Omit } from '@material-ui/types';
+  +import { DistributiveOmit } from '@material-ui/types';
+  ```
+
+### `@material-ui/styles`
+
+#### ThemeProvider
+
+If you are using the utilities from `@material-ui/styles` together with the core components, you should replace the usage of `ThemeProvider` from `@material-ui/styles` with the one exported from `@material-ui/core/styles`. This way, the `theme` provided in the context will be available in both the styling utilities exported from `@material-ui/styles`, like `makeStyles`, `withStyles` etc. and the core components.
```

Comment:
"core components" usage. As long as we don't explain what "core components" are, specifically what the "core" modifier does, we shouldn't use that terminology. Because right now one could think "components from `@material-ui/core`" but that would exclude lab.

Reply:
```suggestion
If you are using the utilities from `@material-ui/styles` together with the `@material-ui/core`, you should replace the usage of `ThemeProvider` from `@material-ui/styles` with the one exported from `@material-ui/core/styles`. This way, the `theme` provided in the context will be available in both the styling utilities exported from `@material-ui/styles`, like `makeStyles`, `withStyles` etc. and the Material-UI components.
```